app/views/content/is-teaching-right-for-me/languages/_article.html.erb

```diff
@@ -0,0 +1,128 @@
+<div class="row inset">
+  <section class="col col-720">
+    <p>
+      A modern languages teacheer opens young minds to the languages and cultures around the world. </p>
+      <p>As we strive to achieve our vision for a 'Global Britain' mutual inter-cultural understanding and insights through the fostering of a shared language, is vital.</p>
```

Comment:
@caitlin-reidy I don't really understand what this sentence means? They're not teaching a shared language they're teaching the language of another country? I'd delete it as I'm not sure what it adds?